tests/_files/conftest.py

```diff
@@ -65,7 +65,7 @@ async def animated_sticker_set(bot):
 
 
 @pytest.fixture(scope="session")
-async def audio(bot, chat_id, aiolib):
```

Comment:
these were from when I was testing a library similar to `pytest-asyncio-cooperative`, i.e. running the tests themselves concurrently